pkg/testutils/mock_config.go

```diff
@@ -31,6 +31,21 @@ func NewFakeClusterConfig(cfgMap *v1.ConfigMap) (*virtconfig.ClusterConfig, cach
 	return virtconfig.NewClusterConfig(configMapInformer, crdInformer, kubeVirtInformer, namespace), configMapInformer, crdInformer, kubeVirtInformer
 }
 
+func NewFakeClusterConfigWithCPUArch(cfgMap *v1.ConfigMap, CPUArch string) (*virtconfig.ClusterConfig, cache.SharedIndexInformer, cache.SharedIndexInformer, cache.SharedIndexInformer) {
```

Comment:
You can use `NewFakeClusterConfigWithCPUArch` in `NewFakeClusterConfig` similar to the `NewCluster*` functions.